### PR TITLE
Introducing E2E Commons

### LINKED ISSUE #64: Simplify maintenance of e2e tests configurations
After merging separate projects to the monorepo we have duplicated configuration of the network and bridges.

**Configuration for the oracle**
  - [script to setup networks, deploy contracts and run bridges](https://github.com/poanetwork/tokenbridge/blob/master/oracle-e2e/run-tests.sh)
  - [bridge and supplementaries configuration](https://github.com/poanetwork/tokenbridge/blob/master/oracle-e2e/docker-compose.yml)
  - [contracts configuration](https://github.com/poanetwork/tokenbridge/tree/master/oracle-e2e/envs)

**Configuration for the UI**
  - [script to setup networks, deploy contracts and run bridges](https://github.com/poanetwork/tokenbridge/blob/master/ui-e2e/run-tests.sh)
  - [bridge and supplementaries configuration](https://github.com/poanetwork/tokenbridge/blob/master/ui-e2e/docker-compose.yml)
  - [contracts configuration](https://github.com/poanetwork/tokenbridge/tree/master/ui-e2e/envs)

It seems logical that the configuration of e2e tests for UI (for example) could inherits the configuration of the e2e tests of the oracle. Otherwise if some changes of the contracts configuration happen it should be reflected in both e2e tests configuration. Similar is for the change on the oracle side - if something new appears (e.g. new configuration parameter or new worker) it will be necessary to modify both e2e tests directories.

Consider build a pipeline of e2e tests configuration in order to simplify the e2e tests maintenance.

### PR DESCRIPTION
**Pull request by rzadp**
- Fixes #64

#### Work done

- Merged common `oracle-e2e` and `ui-e2e` `.env` files into one set
- Merged `docker-compose.yml` files into one
  - Renamed `bridge`, `bridge-erc20` and `bridge-erc20-native` docker services to `oracle`, `oracle-erc20` and `oracle-erc20-native`
- Extracted logic regarding spinning up e2e environment into one script
- Merged script regarding deploying e2e contracts into one script
  - Switched from JS script using shelljs to a shell script
- Merged constants from `oracle-e2e/config.json` and `ui-e2e/constants.json` into `e2e-commons/constants.json`
- Extracted helpers used to mine test blocks into e2e-commons
- Removed LICENCE from `oracle-e2e`

#### Ideas for further improvement
- #85

### CODE REVIEW

**Comment by rzadp** [comment]
@akolotov @patitonar Hey, do you have any questions or comments so far?

File: e2e-commons/down.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+cd $(dirname $0)
+
+if [ $CI ]; then exit $rc; fi
+
+ps | grep node | grep -v grep | awk '{print "kill " $1}' | /bin/bash
```
Comment (akolotov) [inline e2e-commons/down.sh]: could you clarify what `node` process is expected here?

Comment (rzadp) [reply e2e-commons/down.sh]: `node ./scripts/blocks.js &` that can be found in `up.sh`

Comment (akolotov) [reply e2e-commons/down.sh]: thanks!

I would simplify it a bit:
```
ps | grep node | grep -v grep | awk '{print $1}' | xargs kill
```

**Comment by akolotov** [comment]
please wait for @patitonar approve before merge

**Pull request merged**